src/scripts/content/todoist.js

```diff
@@ -1,12 +1,16 @@
 'use strict';
 /* global togglbutton, $ */
 
-togglbutton.render(
-  '.task_item .content:not(.toggl)',
-  { observe: true },
-  elem => {
-    const container = $('.text', elem);
+const EDITOR_ID = 'editor';
+const TASK_ELEMENT = '.content';
+const TOGGL_BUTTON = '.toggl-button';
 
+function insertTogglButton (event) {
+  const { target } = event;
+  const shouldInsert = target.matches(TASK_ELEMENT) && !target.querySelector(TOGGL_BUTTON);
```

Comment:
This doesn't work in all cases.  The problem is that the target will report the deepest DOM node that is hovered, but this is only checking for one particular match.  For example, if you put the mouse over the priority indication / checkbox circles on the left and move the mouse through the list, the toggl button does not appear.

I think this should look through the parents and find one that matches `.task`, the class used on the `<li>` for every task.  Perhaps use another variable, like `var task = null;` that gets set if a `.task` is found in the parents.

Reply:
I also don't think that the `!target.querySelector(TOGGL_BUTTON)` part should be necessary.

I think it might also be good to avoid creating the button if it is already a child of the task element.  Perhaps this isn't necessary, because I'm not seeing duplicated toggl buttons.  However, it seems to me like a fair bit of work is being done whenever `shouldInsert` is `true`, that I think could be avoided if it already exists.

Reply:
`target.querySelector(TOGGL_BUTTON)` checks just that if the list item already has the toggl button inserted into it. Removing it causes multiple instances to be added to a task row.